### PR TITLE
Updated blurb

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Comment by Brandoncharleskhoo1** [comment]
Updated blurb for Cohort Overview page

**Review by Leoaqr — APPROVED** [review]

**Review by Brandoncharleskhoo1 — COMMENTED** [review]
Fixed the Planhat name

File: docs/data/destinations/planhat-event-streaming.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Planhat Event Streaming
+description: Stream Amplitude events to Planhat.
+status: new
+---
+
+!!!beta "This feature is in open beta"
+
+    This feature is in open beta and is in active development. Contact the [Planhat support team]([https://userlist.com/](https://www.google.com/url?q=https://support.planhat.com/en/&sa=D&source=docs&ust=1681362728518389&usg=AOvVaw0HZBKn1FHVmfkZ7_tigFKN)) for support with this integration.
```
Comment (Leoaqr) [inline docs/data/destinations/planhat-event-streaming.md:9]: probably get rid of `userlist` info here?

File: docs/data/destinations/planhat-event-streaming.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- You must enable this integration in each Amplitude project you want to use it in.
+- You need to be an admin of a paid Planhat plan to enable this integration.
+- Amplitude matches the **user_id** to the End User **externalId** within Planhat for associated events. If a user with that **externalId** doesn't exist within Planhat, a user with that **externalId** is created. Make sure that the Amplitude **user_id** field matches the Planhat user **externalId** field to avoid user duplication.
+- Relevant limits for Planhat events are:
+  - Maximum size of event data: 1 MB
+- Amplitude sends all user, event, and group properties along with the event.
+- Note that because the Planhat Analytics endpoint is a “data sink”, it will always return status 200 (OK) even when there is an error in the payload. See Planhat’s [API documentation](https://docs.planhat.com/#response_codes) for details.
+
+## Setup
+See the [Planhat documentation](https://www.google.com/url?q=http://support.planhat.com/en/articles/7181975-setting-up-the-amplitude-integration%23h_87f475e94b&sa=D&source=docs&ust=1681362614710965&usg=AOvVaw2y7G6XhmQHzvEoK8v9w8fU) for more details.
```
Comment (Leoaqr) [inline docs/data/destinations/planhat-event-streaming.md:28]: Update the url with: https://support.planhat.com/en/articles/7181975-setting-up-the-amplitude-integration#h_87f475e94b

File: docs/data/destinations/planhat-event-streaming.md (hunk 1 above)
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/planhat-event-streaming.md]: Fixed it.

**Review by Leoaqr — APPROVED** [review]

**Review by bgiori — APPROVED** [review]

**Review by justin-fiedler — APPROVED** [review]

**Pull request merged**